### PR TITLE
Upgrade for UMD

### PR DESCRIPTION
**Pull request by kmalakoff**
I've made a few modifications to your library to make it compatible with UMD style loading plus some other changes.

- Added umd wrapper for component
- Removed need for an id
- Added chartOptions and configure customization hooks.

Let me know if you are interested in pulling these into your library or if you would like to discuss.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      d3.select(this.refs.svg.getDOMNode()).datum(this.props.datum).call(this.chart);
+
+      nv.utils.windowResize(this.chart.update);
+      return this.chart;
+    });
+  }
+
+  componentDidUpdate() {
+    !this.chart || d3.select(this.refs.svg.getDOMNode()).datum(this.props.datum).call(this.chart);
```
Comment (topicus) [inline index.js]: I remember that sometimes, with some specific options (I don't remember which ones), this kind of update doesn't reflect changes properly. Said that, I will apply this update and if some bug come up then we could report it to the NVD3 guys.

**Comment by topicus** [comment]
This is great! I had plans to move the code to ES6 and you did it. I will take a look at the code and if everything works, merge this.

**Comment by kmalakoff** [comment]
Awesome! Let me know when it is merged and upgraded on npm.

Even though I upgraded it to ES6 syntax, I tried not to change the logic so if there are problems with the options, they are likely to remain. There is the configure hook so users can set up things themselves too. 

Eventually, we could write some tests to and validation on the library, but for now, we can just be pragmatic and make sure it works for us.

**Comment by topicus** [comment]
Working perfect. Merging. Thanks!

**Comment by topicus** [comment]
BTW, I've just update the package in the npm registry.

**Comment by kmalakoff** [comment]
Awesome! I've deleted my repo and updated my project.

**Pull request merged**